src/storr.c

```diff
@@ -0,0 +1,34 @@
+#include <R.h>
+#include <R_ext/Rdynload.h>
+#include <Rinternals.h>
+#include <Rversion.h>
+
+SEXP Cread_text_file(SEXP path, SEXP nchar) {
+  FILE *fp;
+  fp = fopen(CHAR(asChar(path)), "rb");
+  if (fp == NULL) {
+    return R_NilValue;
```

Comment:
Throw with `Rf_error()` to satisfy the storr trait.  All we need is

```c
Rf_error("File %s does not exist", path);
```

Reply:
aa11b1236e5ffff82d5b9128157d4527d0154368. I'm learning a lot about R internals here.